### PR TITLE
[fix] google videos engine

### PR DESCRIPTION
**Pull request by Venca24**

### CODE REVIEW

**Review by joshu9h — APPROVED** [review]
Thanks

**Comment by kvch** [comment]
Thank you for your contribution. I have tested the branch locally. Result thumbnails are not loaded.

missing-img

Do you mind checking this problem?

**Comment by Venca24** [comment]
There were no thumbnails yet before. I tried do my best.

google_videos_stallman

The thumbnails are base64-encoded and given to the results by a JavaScript function.

**Review by kvch — APPROVED** [review]
Thank you!

File: searx/engines/google_videos.py [hunk 1]
```diff
@@ -7,15 +7,16 @@
...
  @results     HTML
  @stable      no
- @parse       url, title, content
+ @parse       url, title, content, thumbnail
 """
 
 from datetime import date, timedelta
 from json import loads
 from lxml import html
+from searx.engines import logger
```
Comment (kvch) [inline searx/engines/google_videos.py]: Do you mind removing the debug logs?

Comment (Venca24) [reply searx/engines/google_videos.py]: Is it inconvenient for possible future debugging of the functionality? Do I need to remove them completely or is it enough just to comment them out?

**Pull request merged**